ci/pipeline

```diff
@@ -122,7 +128,8 @@ def asPullRequest(run: => (utils.SemVer, Option[awsClient.Artifact])): Unit = {
  * @param artifactUrl Url for the saved Marathon artifact.
  * @param sha1 The sah1 checksum of the artifact.
  */
-def updateDcosImage(version: utils.SemVer, artifactUrl: String, sha1: String): Unit = utils.stage("Patch DC/OS Branch") {
+@main
```

Comment:
nit: since it's a main target shouldn't we move it below the `//////////////////// MAIN TARGETS ////////////////////`?